src/boundary_conditions/src/pressure_pinning.C

```diff
@@ -66,11 +69,43 @@ namespace GRINS
     return;
   }
 
-  void PressurePinning::pin_value( libMesh::DiffContext &context, 
+  void PressurePinning::check_pin_location( const libMesh::MeshBase& mesh )
+  {
+    libMesh::MeshBase::const_element_iterator el = mesh.active_local_elements_begin();
+    const libMesh::MeshBase::const_element_iterator end_el = mesh.active_local_elements_end();
+
```

Comment:
But then you're iterating over all active local elements without a boundary test?